backend/src/board.rs

```diff
@@ -230,6 +231,21 @@ impl Board {
         *self.inner().is_fake
     }
 
+    pub fn is_lite(&self) -> bool {
+        static RE: Lazy<Regex> = Lazy::new(|| Regex::new("system76/launch_lite_.*").unwrap());
+        RE.is_match(self.model())
+    }
+
+    pub fn is_1(&self) -> bool {
+        static RE: Lazy<Regex> = Lazy::new(|| Regex::new("system76/launch_1.*").unwrap());
+        RE.is_match(self.model())
+    }
+
+    pub fn is_2(&self) -> bool {
```

Comment:
Why are `launch_heavy_1` and `launch_2` being grouped together here?

Reply:
Because they do the same thing. Production needs to be able to flash heavys as 2s, and 2s as heavys.

Reply:
They both use the same launch 2.0 PCB

Reply:
Makes sense. Might be worth having a comment here mentioning that.